docs/DataDirectory.md

```diff
@@ -0,0 +1,36 @@
+# Indexer Data Directory
+
+The Indexer data directory is the location where the Indexer can store and/or load data needed for runtime operation and configuration. It is a required argument for Indexer daemon operation. Supply it to the Indexer via the `--data-dir` flag.
+
+# Storage Requirements
+
+As of mid-2022, approximately 15GiB for Mainnet.
+
+# Configuration Files
+
+The data directory is the first place to check for different configuration files, for example:
+- `indexer.yml` - Indexer Configuration File
+- `api_config.yml` - API Parameter Enable/Disable Configuration File
+
+# Account Cache
+
+Indexer writers keep an account cache in the data directory. This cache is used during block processing to compute things like the new account balances after processing transactions. Prior to this local cache, the database was queried on each round to fetch the initial account states.
+
+## Read-Only Mode
```

Comment:
If an indexer is started in writer mode and then is re-started in reader mode, will these files be there or will they be deleted by the indexer?

Reply:
They wouldn't be deleted, but wouldn't be updated either.